ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -217,19 +226,13 @@ Create Openvino Models
     END
 
 Model Serving Suite Teardown
-    [Documentation]    Suite teardown steps after testing DSG. It Deletes
-    ...                all the DS projects created by the tests and run RHOSi teardown
-    # Even if kw fails, deleting the whole project will also delete the model
-    # Failure will be shown in the logs of the run nonetheless
-    IF    ${MODEL_CREATED}
-        Clean All Models Of Current User
-    ELSE
-        Log    Model not deployed, skipping deletion step during teardown    console=true
-    END
-    ${projects}=    Create List    ${PRJ_TITLE}
-    Delete Data Science Projects From CLI   ocp_projects=${projects}
-    # Will only be present on SM cluster runs, but keyword passes
-    # if file does not exist
+    [Documentation]    Suite teardown steps after testing DSG.
     Remove File    openshift_ca.crt
     Close All Browsers
     RHOSi Teardown
+
+Model Serving Test Teardown
+    [Documentation]    Test teardown steps after test. It Deletes
+    ...                all the DS projects created by the tests and run RHOSi teardown
```

Comment:
RHOSi teardown too? :thinking:

Reply:
Copypasting like a rookie.
Thanks for noticing!
Fixed in [b1a4d1c](https://github.com/red-hat-data-services/ods-ci/pull/1402/commits/b1a4d1c5afd7abc376bb0f9ef49f64c00be504b0)